tapir/coop/templates/coop/draftuser_list.html

```diff
@@ -30,25 +30,25 @@ <h5 class="card-header d-flex justify-content-between align-items-center">
                     <th>{% translate "Date" %}</th>
                     <th></th>
                     </thead>
-                    {% for u in object_list %}
+                    {% for user in object_list %}
```

Comment:
In that case I think we should specify `draft_user` to not mix it with `tapir_user`?